Fixed configure script compatibility issue.
Applied manually on configure from 2943255ebaf5c7a574e8c14b474fb6cf2562f77a

--- configure.orig	2025-03-28 22:55:28 UTC
+++ configure
@@ -17383,7 +17383,7 @@ ac_compiler_gnu=$ac_cv_c_compiler_gnu
     CXX="$CXX"
   else
     result=`echo "${values_to_check}" | $GREP -Fvx -- "${unwanted_values}" | $GREP -v '^$'`
-    CXX=${result//$'\n'/ }
+    CXX=$(echo "$result" | tr -d '\n')
   fi
 
 
